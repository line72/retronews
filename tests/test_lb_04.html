<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
<blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, <a href="https://example.com/" rel="ugc">example</a> sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur <code>adipiscing elit</code>, sed do eiusmod tempor</p>
<blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</p>
</blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</p>
</blockquote>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>