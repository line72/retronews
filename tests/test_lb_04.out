Lorem ipsum dolor sit amet, consectetur adipiscing elit

> Lorem ipsum dolor sit amet, consectetur adipiscing elit, example
> https://example.com/ sed do eiusmod tempor. Lorem ipsum dolor sit
> amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem
> ipsum dolor sit amet, consectetur `adipiscing elit`, sed do eiusmod
> tempor
> 
>> Lorem ipsum /dolor/ *sit
>> dolor* amet, consectetur adipiscing elit
>> 
>> ------------------------------------------------------------------
>> 
>> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
>> eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing
>> elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet,
>> consectetur adipiscing elit, sed do eiusmod tempor
> 
> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
> eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing
> elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur
> adipiscing elit, sed do eiusmod tempor

Lorem ipsum dolor sit amet, consectetur adipiscing elit